app/src/main/java/com/mobileproto/hireddit/hireddit/reddit/RedditSearcher.java

```diff
@@ -106,7 +101,9 @@ private void getCommentFromKeywords(List<String> keywords) {
                 .appendPath("reddit")
                 .appendPath("search")
                 .appendQueryParameter("q", importantWords)
-                .appendQueryParameter("fields", fields);
+                .appendQueryParameter("fields", fields)
+                .appendQueryParameter("limit", "1000");
```

Comment:
It's the 1000 most recent comments. It's so that we can look further back in time for comments. It doesn't actually affect runtime at all having this number be big.